deps.windows/40-luajit.ps1

```diff
@@ -2,11 +2,15 @@ param(
     [string] $Name = 'luajit',
     [string] $Version = '2.1',
     [string] $Uri = 'https://github.com/luajit/luajit.git',
-    [string] $Hash = '8635cbabf3094c4d8bd00578c7d812bea87bb2d3'
+    [hashtable] $Hashes = @{
+        x64 = '8635cbabf3094c4d8bd00578c7d812bea87bb2d3'
+        x86 = '8635cbabf3094c4d8bd00578c7d812bea87bb2d3'
+        arm64 = '43d0a19158ceabaa51b0462c1ebc97612b420a2e'
+    }
```

Comment:
Will be obsoleted once we update this for all systems.

Reply:
Should be obsoleted by https://github.com/obsproject/obs-deps/commit/96549337dff09177ff75ed480a629b5cb5e7c97f and https://github.com/obsproject/obs-deps/commit/ec04a292eb77c141ac5ae603b5ec92121366b17c (#228), which updated to a newer LuaJIT commit.